README.md

```diff
@@ -8,37 +8,63 @@ CoEpi Cloud API on AWS.
 * **Routing and Load Balancing**: API Gateway
 * **Permissions Management**: IAM
 
-## Build
+## Getting Started
+
+Requirements:
+
+* JDK11 or newer
+
+### Infrastructure Setup
+
+All infrastructure concerns are managed by [Terraform](https://terraform.io/) (v0.12+). You can either download Terraform from their site, or install it with [Choco](https://chocolatey.org/) (Windows), [brew](https://brew.sh) (macOS), or your Linux distribution's package manager.
+
+Next you will want to create an S3 bucket to store your Terraform [remote state](https://www.terraform.io/docs/backends/index.html). This will need to be uniquely named, and should be private and encrypted. A script has been provided to do this for you:
```

Comment:
Recommend to provide the option to use the local state, if the developer just wants a quick start.

Reply:
Makes sense. Should we say that remote state is the preferred option? After all, if you're using the Terraform you must already have an AWS account, and it avoids a messy situation if somebody loses their local tfstate.